components/datetime/src/options/components.rs

```diff
@@ -578,15 +578,15 @@ pub enum TimeZoneName {
     /// Long localized form, without the location (e.g., Pacific Standard Time, Nordamerikanische Westküsten-Normalzeit)
     LongSpecific,
 
-    // UTS-35 fields: O, OOOO
+    // UTS-35 fields: OOOO
     // Per UTS-35: The long localized offset format. This is equivalent to the "OOOO" specifier
-    // Per UTS-35: Short localized offset format (e.g., GMT-8)
-    // This enum variant is combining the two types of fields, as the CLDR specifies the preferred
-    // hour-format for the locale, and ICU4X uses the preferred one.
-    //   e.g.
-    //   https://github.com/unicode-org/cldr-json/blob/c23635f13946292e40077fd62aee6a8e122e7689/cldr-json/cldr-dates-full/main/es-MX/timeZoneNames.json#L13
-    /// Localized offset format, in the locale's preferred hour format. (e.g., GMT-0800),
-    Offset,
+    /// Long localized offset form, e.g. GMT-08:00
+    LongOffset,
+
+    // UTS-35 fields: O
+    // Per UTS-35: Short localized offset format
+    /// Short localized offset form, e.g. GMT-8
+    ShortOffset,
```

Comment:
Note: this is components bag code, which is behind `#[cfg(feature = "datagen")]`

Reply:
Ok? Should I not touch this?

Reply:
It's fine. Just a note.